CHANGELOG.md

```diff
@@ -7,6 +7,14 @@
 
 ## **[Unreleased]**
 
+### Added
+
+### Changed
+
+### Fixed
+
+## 1.0.1 - 2021-01-12
+
```

Comment:
should we add a note here saying what @mark wrote in the description just to make things clear about the "bc break"?

Reply:
Yeah, that would be a good idea 👍

Reply:
Alright, done!